tests/RepositoryForms/Twig/LimitationValueRenderingExtensionTest.php

```diff
@@ -32,8 +32,14 @@ public function getExtensions(\Twig_Environment $twig = null)
         );
 
         $limitationBlockRenderer->setLimitationValueResources([
-            'templates/limitation_value_1.html.twig',
-            'templates/limitation_value_2.html.twig',
+            [
+                'template' => 'templates/limitation_value_1.html.twig',
+                'priority' => 0,
+            ],
+            [
+                'templates/limitation_value_2.html.twig',
+                'priority' => 0,
+            ],
```

Comment:
Since the feature includes parsing of priorities, can you set different priorities here and test them? (unless it's covered elsewhere).